src/Phinx/Console/Command/SeedCreate.php

```diff
@@ -178,11 +178,22 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             ));
         }
 
+        // Command-line option must have higher priority than value from Config
+        $config = $this->getConfig();
+        if (is_null($altTemplate)) {
+            $altTemplate = $config->getSeederTemplateFile();
+            if (!is_null($altTemplate) && !is_file($altTemplate)) {
+                throw new InvalidArgumentException(sprintf(
+                    'The template file "%s" from config does not exist',
```

Comment:
` is used for code fencing, not "

Reply:
> ` is used for code fencing, not "

Oh, sorry, I copy-pasted it from line 176 by @Firtzberg from 2021-01-13, and I don't even know what does that fencing means... I will read about it for sure... but for this code should I replace double quotes with ` in the whole method or just in my code?

Reply:
Ok, replaced " with ` here and in the test